### PR TITLE
Feat: Added Navigation

### PR DESCRIPTION
**Pull request by anamkaa**
Added variations of navigation
- navigation with icon
- navigation with icon and text
- navigation with text
- navigation dark

Screenshot (265)

### CODE REVIEW

File: comp-layouts/navigation.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (Sameony) [inline comp-layouts/navigation.html]: Change the title of the document to something relevant

**Review by Sameony — COMMENTED** [review]
Good clear code. Do clean the white spaces with prettier/beautify if those single empty lines after the nav bar items are not intentional (eg: line 38, 59, 73, 85)

**Review by SimranjitKaurMaan — APPROVED** [review]
Looks good.

File: comp-layouts/navigation.html (hunk 1 above)
Comment (anamkaa) [reply comp-layouts/navigation.html]: Done as suggested.

**Pull request merged**